src/operator/rnn-inl.h

```diff
@@ -1378,9 +1378,17 @@ class RNNOp {
                                                 &reserve_space_byte_));
       workspace_size_ = workspace_byte_ / sizeof(DType);
       // Allocate the reserve space
-      reserve_space_ = Storage::Get()->Alloc(reserve_space_byte_, Context::GPU(s->dev_id));
+      reserve_space_ = Storage::Get()->Alloc(reserve_space_byte_, Context::GPU(s->dev_id)
```

Comment:
Did you measure the overhead of always passing the string? Having these macros is not easy to maintain. 
If the overhead is neglegible it's probably cleaner to alway pass the name

Reply:
In my opinion the overhead is very difficult to measure. The reason why I added those flags is because there might be cases where people just want to work with a *clean* version of MXNet with nothing added and hence do not need to worry about the overhead.